content/200-orm/200-prisma-client/150-using-raw-sql/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Write Your Own SQL'
```

Comment:
let's use regular sentence casing for this to be consistent with the other sidenav items?

```suggestion
title: 'Write your own SQL'
```